docker/ci/DOCKER.md

```diff
@@ -0,0 +1,25 @@
+# Docker
+
+The Docker CI is used to run linting and tests designed for a CI pipeline. Github workflows have been added to .github/workflows that will automatically run linting and tests for only the plugin.
+
+These tests use OpenProject Core `v12.0.1` by default. To change this, go to `Dockerfile` and change `OPENPROJECT_BRANCH` to your desired version number.
```

Comment:
We could say _"use a recent OpenProject Core production release by default"_, putting actual version numbers in the documentation will make it to rot away quicker, since the next person updating the `Dockerfile` will most certainly forget to update the Markdown as well.